test/jdk/java/awt/Dialog/FileDialogIconTest/FileDialogIconTest.java

```diff
@@ -38,7 +39,7 @@
 
 /*
  * @test
- * @bug 4035189
+ * @bug 6425126
  * @summary Test to verify that PIT File Dialog icon not matching with
```

Comment:
Do we still need PIT build/PIT file Dailog in summary? The main bug seems to be in solaris.